pkg/storaged/client.js

```diff
@@ -1484,4 +1493,40 @@
     }
 };
 
+client.in_anaconda_mode = () => !!client.anaconda;
+
+client.strip_mount_point_prefix = (dir) => {
+    const mpp = client.anaconda?.mount_point_prefix;
+
+    if (dir && mpp) {
+        if (dir.indexOf(mpp) != 0)
+            return false;
+
+        dir = dir.substr(mpp.length);
+        if (dir == "")
+            dir = "/";
+    }
+
+    return dir;
+};
+
+client.add_mount_point_prefix = (dir) => {
+    const mpp = client.anaconda?.mount_point_prefix;
+    if (mpp) {
+        if (dir == "/")
+            dir = mpp;
+        else
+            dir = mpp + dir;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231211-144024-c4e15dfd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)